src/analytics/types.ts

```diff
@@ -58,6 +58,13 @@ export enum Action {
   CreateDiscountClick = "CREATE_DISCOUNT_CLICK",
   CreateQuicklinkClick = "CREATE_QUICKLINK_CLICK",
   ApplyFinanceClick = "APPLY_FINANCE_CLICK",
+
+  // Checkout 2.0
+  EmailInput = "EMAIL_INPUT",
+  PhoneInput = "PHONE_INPUT",
+  LoginPasswordInput = "LOGIN_PASSWORD_INPUT",
+  SignUpPasswordInput = "SIGNUP_PASSWORD_INPUT",
+  EmailOtpInput = "EMAIL_OTP_INPUT",
 }
 
 export enum Title {
```

Comment:
In place where we have repeatability like in case of subscription related titles, in other languages we do have nested enums. In TS, we don't have nested enums directly but we can get pseudo nested enums by making child enums and then wrapping them in a const, so, in this case it would be something like:

`
const Title = {
Subscriptions,
....}

enum Subscriptions {
Home = "HOME",
...}
`